docs/src/tutorial/clibraries.rst

```diff
@@ -250,19 +316,31 @@ to
                   libraries=["calg"])
         ])
 
-If it is not installed in a 'normal' location, users can provide the
+Now we should be able to build the project using::
+
+    $ python setup.py build_ext -i 
+
+If the `libcalg` is not installed in a 'normal' location, users can provide the
 required parameters externally by passing appropriate C compiler
 flags, such as::
 
     CFLAGS="-I/usr/local/otherdir/calg/include"  \
     LDFLAGS="-L/usr/local/otherdir/calg/lib"     \
         python setup.py build_ext -i
 
+
+
+Before we run the module, we also need to make sure that `libcalg` is in the LD_LIBRARY_PATH, e.g. by 
+running::
```

Comment:
"running" -> "setting", and better mention that this is an "environment variable".